README.md

```diff
@@ -1 +1,65 @@
-# SAMPL9
\ No newline at end of file
+# The SAMPL9 Blind Predicton Challenges for Computational Chemistry
+
+
+Challenge details, inputs, and results from the SAMPL8 series (phase) of challenges. Each individual SAMPL8 challenge may be broken up into multiple stages. 
+
+See the [SAMPL website](https://samplchallenges.github.io/) for information on the Statistical Assessement of the Modeling of Proteins and Ligands (SAMPL) series of challenges as a whole. This repository focuses specifically on the SAMPL9 sreies of challenges. Additionally, see the [SAMPL community](https://zenodo.org/communities/sampl/?page=1&size=20) on Zenodo for content related to the SAMPL series of challenges. If you wish to use Zenodo to post our presentaiton slides, datasets, and/or other SAMPL related material, and get a DOI for them, please upload to the community [here](https://zenodo.org/communities/sampl/?page=1&size=20) so that your content will be listed. 
+
+Because these files are available publicly, we have no record of who downloads them. Therefore, you should sign up for notifications. Specifically, if you want to receive updates if we uncover any problems, it is imperative that you either (a) [sign up for the SAMPL e-mail list](https://mailchi.mp/e36018629725/sampl9-sign-ups), or (b) sign up for notifications of changes to this GitHub repository (the Watch button, above); ideally you would do both. 
```

Comment:
The sign-up URL doesn't exist yet. I need to create or we should modify the link; let me check.